src/tests/unittests/configuration_helper/adapters/test_hdawg8_instrument_adapter.py

```diff
@@ -193,6 +193,14 @@ def test_stop(self):
             dawg.assert_has_calls([call().stop_awg(1)])
             hdawg_adapter.instrument.close()
 
+    def test_close_instrument(self):
```

Comment:
The InstrumentAdapter is an abc, I guess that is the reason this test is in the hdawg8 instrument adapter. It would be nicer to put the test in the `test_instrument_adapter.py` though.

Reply:
I thought so as well so I will move it, or create another test in `test_instrument_adapter.py`